cpp/ql/lib/semmle/code/cpp/ir/implementation/aliased_ssa/internal/SSAConstruction.qll

```diff
@@ -981,7 +981,7 @@ module DefUse {
   }
 }
 
-predicate canReuseSsaForMemoryResult(Instruction instruction) {
+deprecated predicate canReuseSsaForMemoryResult(Instruction instruction) {
```

Comment:
I figured that out when the identical files check failed. This is still a draft PR, so thanks for the early review 👍

All of this will hopefully be fixed with parameterized modules 🤞